wlutil/busybox-config

```diff
@@ -838,27 +838,27 @@ CONFIG_WATCHDOG=y
 #
 # Networking Utilities
 #
-CONFIG_FEATURE_IPV6=y
+# CONFIG_FEATURE_IPV6 is not set
 # CONFIG_FEATURE_UNIX_LOCAL is not set
-CONFIG_FEATURE_PREFER_IPV4_ADDRESS=y
+# CONFIG_FEATURE_PREFER_IPV4_ADDRESS is not set
 # CONFIG_VERBOSE_RESOLUTION_ERRORS is not set
 # CONFIG_FEATURE_TLS_SHA1 is not set
-CONFIG_ARP=y
-CONFIG_ARPING=y
+# CONFIG_ARP is not set
+# CONFIG_ARPING is not set
 # CONFIG_BRCTL is not set
 # CONFIG_FEATURE_BRCTL_FANCY is not set
 # CONFIG_FEATURE_BRCTL_SHOW is not set
-CONFIG_DNSD=y
-CONFIG_ETHER_WAKE=y
+# CONFIG_DNSD is not set
+# CONFIG_ETHER_WAKE is not set
 # CONFIG_FTPD is not set
 # CONFIG_FEATURE_FTPD_WRITE is not set
 # CONFIG_FEATURE_FTPD_ACCEPT_BROKEN_LIST is not set
 # CONFIG_FEATURE_FTPD_AUTHENTICATION is not set
 # CONFIG_FTPGET is not set
 # CONFIG_FTPPUT is not set
 # CONFIG_FEATURE_FTPGETPUT_LONG_OPTIONS is not set
-CONFIG_HOSTNAME=y
-CONFIG_DNSDOMAINNAME=y
+# CONFIG_HOSTNAME is not set
```

Comment:
I suggest reenabling `CONFIG_HOSTNAME` or removing the line in `/etc/inittab` that updates `/etc/hostname`.

It's for an almost purely cosmetic reason, but it does allow the system to boot without visible errors.